Snapshot releases (Mac and Linux versions) on MacOS are not properly loading

When opening the Mac OS (.dmg) snapshot release, it doesn't load OpenRefine any further than the logo, as seen in this screenshot:
![image](https://user-images.githubusercontent.com/9881645/162605130-f818968a-eed4-44df-9499-273de95bfdf2.png)

When I try with the Linux download, that one doesn't even open a new browser tab at all.

### Expected Behavior
OpenRefine should fully load.

### Context
Terminal log when trying to start the MacOS .dmg (pretty minimal):
```
Sandras-MacBook-Air:~ fokky$ /Applications/OpenRefine\ snapshot\ 20220408.app/Contents/MacOS/JavaAppLauncher ; exit;
08:15:29.649 [            refine_server] Starting Server bound to '127.0.0.1:3333' (0ms)
08:15:29.717 [            refine_server] Initializing context: '/' from '/Applications/OpenRefine snapshot 20220408.app/Contents/Resources/webapp' (68ms)
08:15:31.931 [                   refine] Starting OpenRefine 3.6-SNAPSHOT [0ba3b92]... (2214ms)
08:15:31.937 [                   refine] initializing FileProjectManager with dir (6ms)
08:15:31.937 [                   refine] /Users/fokky/Library/Application Support/OpenRefine (0ms)
```

When I try to open the Linux version of the snapshot release, that one doesn't even start up a new browser tab. Terminal log for that one is:

```
miniMac:~ fokky$ /Users/fokky/Desktop/openrefine-3.6-SNAPSHOT\ 3/refine ; exit;
Using refine.ini for configuration
cat: refine.ini: No such file or directory
-------------------------------------------------------------------------------------------------
You have 812M of free memory.
Your current configuration is set to use 1024M of memory.
OpenRefine can run better when given more memory. Read our FAQ on how to allocate more memory here:
https://docs.openrefine.org/manual/installing#increasing-memory-allocation
-------------------------------------------------------------------------------------------------
ls: server/target/lib: No such file or directory
\nCould not find Maven locally, starting download for Maven ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 8469k  100 8469k    0     0  10.7M      0 --:--:-- --:--:-- --:--:-- 10.7M
/Users/fokky/Desktop/openrefine-3.6-SNAPSHOT 3/refine: line 294: cd: main/webapp: No such file or directory
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.300 s
[INFO] Finished at: 2022-04-10T09:26:43+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (/Users/fokky/Desktop). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.426 s
[INFO] Finished at: 2022-04-10T09:26:47+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (/Users/fokky/Desktop). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException


Error: Could not find or load main class com.google.refine.Refine
Caused by: java.lang.ClassNotFoundException: com.google.refine.Refine
logout

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

### Versions<!--  (please complete the following information)-->
 - Operating System: MacOS 10.13.6, MacOS 12.2.1
 - Browser Version: Chrome
 - JRE or JDK Version: n/a?
 - OpenRefine: 3.6 snapshot release

### Files used

I just tried with the April 8 and April 10 snapshot releases, but the problem has been around for longer (several weeks?).

I heard from a Linux user (via the Structured Data on Commons Telegram group) that they have the same issue of OpenRefine snapshot releases not loading for them.

--- a/.github/workflows/snapshot_release.yml
+++ b/.github/workflows/snapshot_release.yml
@@ -174,6 +174,9 @@ jobs:
       run: |
         mvn jacoco:report coveralls:report -DrepoToken=${{ env.COVERALLS_TOKEN }} -DpullRequest=${{ github.event.number }} -DserviceName="GitHub Actions" -DserviceBuildNumber=${{ env.GITHUB_RUN_ID }} -Dbranch=master
 
+    - name: Install webapp dependencies
+      run: cd main/webapp && npm install
+
     - name: Generate dist files
       run: mvn package -DskipTests=true
 

